panel/src/helpers/isComponent.js

```diff
@@ -6,3 +6,12 @@
 export default (name) => {
 	return typeof window.Vue.options.components[name] === "function";
 };
+
+/**
+ * Checks if the coponent is an instance of Vue
+ * @param {object} component
+ * @returns {bool}
+ */
+export function isVueComponent(component) {
+	return typeof component?.render === "function";
```

Comment:
Does this work when components set a template instead of defining a render function?

Reply:
Hmm no and this doesn't seem to work as I thought (even though it's the suggested way when googling).

I think there are two ways:
- check for `component._isVue`
- check for `component.$el`

I would lean towards the second but not as `isVueComponent` helper but directly just replacing the line in `DropdownContent`